covasim/webapp/cova_app.js

```diff
@@ -34,9 +34,11 @@ const PlotlyChart = {
 const interventionTableConfig = {
     social_distance: {
         formTitle: "Physical distancing",
-        fields: [{key: 'start', type: 'number', label: 'Start day'},
-            {key: 'end', type: 'number', label: 'End day'},
-            {label: 'Effectiveness', key: 'level', type: 'select', options: [{label: 'Aggressive (80%)', value: 'aggressive'}, {label: 'Moderate (50%)', value: 'moderate'}, {label: 'Mild (20%)', value: 'mild'}]}],
+        fields: [
+            {key: 'start', type: 'number', label: 'Start day', tooltip: 'Start day'},
+            {key: 'end', type: 'number', label: 'End day', tooltip: 'End day'},
+            {key: 'level', type: 'number', label: 'Effectiveness', tooltip: 'Effectiveness', min: 0, max: 100}
```

Comment:
Tooltip for effectiveness should be more descriptive, (and I'm guessing what this actually means) but along the lines of "Percent reduction in contacts" (or what it actually happens to be ;-))